lib/array-store.js

```diff
@@ -9,15 +9,14 @@ var ArrayStore = function(datasetId, data) {
   this.topic = {};
   this.subscription = {};
   this.datasetId = datasetId;
-}
+};
 
-ArrayStore.prototype.create = function(object, ts) {
+ArrayStore.prototype.create = function(object) {
   var self = this;
   var deferred = q.defer();
   setTimeout(function() {
     object.id = shortid.generate();
     self.data.push(object);
-    console.log('Created object:', object);
```

Comment:
`eslint` rules only emit a warning for `console` calls, which still allow the build to pass. Do we want to remove these debug calls here?